sources/dynamodb/README.md

```diff
@@ -1,54 +1,100 @@
-# HarbourBridge: Turnkey DynamoDB-to-Spanner Evaluation
+# HarbourBridge: DynamoDB-to-Spanner Evaluation and Migration
 
-HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
-using data from an existing PostgreSQL or MySQL or DynamoDB database. This
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and migration,
+using data from an existing database. This
 README provides details of the tool's DynamoDB capabilities. For general
-HarbourBridge information see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation).
+HarbourBridge information see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-spanner-evaluation-and-migration).
 
 ## Example DynamoDB Usage
 
-The following examples assume `harbourbridge` alias has been setup as
```

Comment:
I think it is worth noting the alias and referring to the section in the main README. How about:

The following examples assume a `harbourbridge` alias has been setup as described in the Installing HarbourBridge section of the main README (with link).

Reply:
And same comment for the mysql/README.md and postgres/README.md.

Reply:
I went back and forth on this, thanks for the input. Have now added back "noting the alias" text.